lib/ood_core/batch_connect/templates/vnccontainer.rb

```diff
@@ -0,0 +1,248 @@
+require "ood_core/refinements/hash_extensions"
+
+module OodCore
+  module BatchConnect
+    class Factory
+      using Refinements::HashExtensions
+
+      # Build the VNC template from a configuration
+      # @param config [#to_h] the configuration for the batch connect template
+      def self.build_vnccontainer(config)
```

Comment:
```suggestion
      def self.build_vnc_container(config)
```

I think we should have a word separator here. I tested this and it's OK, just have to rename the file too.